src/components/UploadingJobsDisplay/UploadingJobsDisplay.js

```diff
@@ -452,6 +458,69 @@ export class UploadingJobsDisplay extends Component {
     this.setState({ renderLeaveModal: false });
   };
 
+  // TODO: this is temporary way (will be changed/removed) to initiate loading of MARC BIB files (UIDATIMP-185)
+  loadMarcRecords = async menu => {
+    const { stripes: { okapi } } = this.props;
+    const { uploadDefinition } = this.context;
+
+    try {
+      await loadMarcRecords({
+        uploadDefinitionId: uploadDefinition.id,
+        // jobProfileId for now is hardcoded to point to the default job profile (MODSOURMAN-113)
+        // later jobProfileId will be picked through UI
+        jobProfileId: '22fafcc3-f582-493d-88b0-3c538480cd83',
+        okapi,
+      });
+    } catch (error) {
+      menu.onToggle();
+      this.calloutRef.current.sendCallout({
+        type: 'error',
+        message: <FormattedMessage id="ui-data-import.communicationProblem" />,
+      });
+
+      this.setState({ loadingRecordsInProgress: false });
+      console.error(error); // eslint-disable-line no-console
+    }
+  };
+
+  // TODO: this is temporary way (will be changed/removed) of deciding whether to allow to initiate
+  // the process of loading MARC BIB files (UIDATIMP-185)
+  // later the button and logic for showing will be removed
+  renderLoadMarcButton = menu => {
+    const {
+      files = [],
+      loadingRecordsInProgress,
```

Comment:
It seems, recordsLoadingInProgress sounds better

Reply:
Agree. Gave it a long thought and think that it may be better. Will change.